addons/mail/wizard/mail_compose_message.py

```diff
@@ -528,3 +528,7 @@ def render_message(self, cr, uid, wizard, res_id, context=None):
 
     def generate_email_for_composer(self, cr, uid, template_id, res_id, context=None):
         return self.generate_email_for_composer_batch(cr, uid, template_id, [res_id], context)[res_id]
+
+    # Get mention user list (@ Functionality)
+    def get_mention_users(self, cr, uid, search_str, limit, context=None):
+        return self.pool.get('res.users').search_read(cr, uid, ['|', ('login', 'ilike', search_str), ('name', 'ilike', search_str)], ['partner_id', 'login'], limit=limit, context=context)
```

Comment:
@rim-odoo 
But we need partner_id, name and login fields.
while name_search just return user id and name.